************* Module convert_numbers
convert_numbers.py:10:0: W0311: Bad indentation. Found 2 spaces, expected 4 (bad-indentation)
convert_numbers.py:11:0: W0311: Bad indentation. Found 2 spaces, expected 4 (bad-indentation)
convert_numbers.py:12:0: W0311: Bad indentation. Found 4 spaces, expected 8 (bad-indentation)
convert_numbers.py:13:0: W0311: Bad indentation. Found 6 spaces, expected 12 (bad-indentation)
convert_numbers.py:14:0: W0311: Bad indentation. Found 8 spaces, expected 16 (bad-indentation)
convert_numbers.py:15:0: W0311: Bad indentation. Found 10 spaces, expected 20 (bad-indentation)
convert_numbers.py:16:0: W0311: Bad indentation. Found 8 spaces, expected 16 (bad-indentation)
convert_numbers.py:17:0: W0311: Bad indentation. Found 10 spaces, expected 20 (bad-indentation)
convert_numbers.py:18:0: W0311: Bad indentation. Found 2 spaces, expected 4 (bad-indentation)
convert_numbers.py:19:0: W0311: Bad indentation. Found 4 spaces, expected 8 (bad-indentation)
convert_numbers.py:20:0: W0311: Bad indentation. Found 4 spaces, expected 8 (bad-indentation)
convert_numbers.py:21:0: W0311: Bad indentation. Found 2 spaces, expected 4 (bad-indentation)
convert_numbers.py:24:0: W0311: Bad indentation. Found 2 spaces, expected 4 (bad-indentation)
convert_numbers.py:25:0: W0311: Bad indentation. Found 2 spaces, expected 4 (bad-indentation)
convert_numbers.py:26:0: W0311: Bad indentation. Found 4 spaces, expected 8 (bad-indentation)
convert_numbers.py:27:0: W0311: Bad indentation. Found 4 spaces, expected 8 (bad-indentation)
convert_numbers.py:28:0: W0311: Bad indentation. Found 2 spaces, expected 4 (bad-indentation)
convert_numbers.py:31:0: W0311: Bad indentation. Found 2 spaces, expected 4 (bad-indentation)
convert_numbers.py:32:0: W0311: Bad indentation. Found 2 spaces, expected 4 (bad-indentation)
convert_numbers.py:33:0: W0311: Bad indentation. Found 2 spaces, expected 4 (bad-indentation)
convert_numbers.py:34:0: W0311: Bad indentation. Found 4 spaces, expected 8 (bad-indentation)
convert_numbers.py:35:0: W0311: Bad indentation. Found 4 spaces, expected 8 (bad-indentation)
convert_numbers.py:36:0: W0311: Bad indentation. Found 2 spaces, expected 4 (bad-indentation)
convert_numbers.py:39:0: W0311: Bad indentation. Found 2 spaces, expected 4 (bad-indentation)
convert_numbers.py:40:0: W0311: Bad indentation. Found 4 spaces, expected 8 (bad-indentation)
convert_numbers.py:41:0: W0311: Bad indentation. Found 4 spaces, expected 8 (bad-indentation)
convert_numbers.py:43:0: W0311: Bad indentation. Found 2 spaces, expected 4 (bad-indentation)
convert_numbers.py:44:0: W0311: Bad indentation. Found 2 spaces, expected 4 (bad-indentation)
convert_numbers.py:46:0: W0311: Bad indentation. Found 2 spaces, expected 4 (bad-indentation)
convert_numbers.py:47:0: C0303: Trailing whitespace (trailing-whitespace)
convert_numbers.py:48:0: W0311: Bad indentation. Found 2 spaces, expected 4 (bad-indentation)
convert_numbers.py:49:0: W0311: Bad indentation. Found 4 spaces, expected 8 (bad-indentation)
convert_numbers.py:50:0: W0311: Bad indentation. Found 4 spaces, expected 8 (bad-indentation)
convert_numbers.py:52:0: W0311: Bad indentation. Found 2 spaces, expected 4 (bad-indentation)
convert_numbers.py:53:0: W0311: Bad indentation. Found 2 spaces, expected 4 (bad-indentation)
convert_numbers.py:54:0: W0311: Bad indentation. Found 4 spaces, expected 8 (bad-indentation)
convert_numbers.py:55:0: W0311: Bad indentation. Found 4 spaces, expected 8 (bad-indentation)
convert_numbers.py:56:0: W0311: Bad indentation. Found 4 spaces, expected 8 (bad-indentation)
convert_numbers.py:58:0: W0311: Bad indentation. Found 2 spaces, expected 4 (bad-indentation)
convert_numbers.py:60:0: W0311: Bad indentation. Found 2 spaces, expected 4 (bad-indentation)
convert_numbers.py:61:0: W0311: Bad indentation. Found 2 spaces, expected 4 (bad-indentation)
convert_numbers.py:63:0: W0311: Bad indentation. Found 2 spaces, expected 4 (bad-indentation)
convert_numbers.py:64:0: W0311: Bad indentation. Found 4 spaces, expected 8 (bad-indentation)
convert_numbers.py:65:0: W0311: Bad indentation. Found 4 spaces, expected 8 (bad-indentation)
convert_numbers.py:9:0: C0116: Missing function or method docstring (missing-function-docstring)
convert_numbers.py:9:17: W0621: Redefining name 'archivo' from outer scope (line 43) (redefined-outer-name)
convert_numbers.py:10:2: W0621: Redefining name 'numeros' from outer scope (line 46) (redefined-outer-name)
convert_numbers.py:12:31: W0621: Redefining name 'f' from outer scope (line 63) (redefined-outer-name)
convert_numbers.py:18:2: W0702: No exception type(s) specified (bare-except)
convert_numbers.py:12:9: W1514: Using open without explicitly specifying an encoding (unspecified-encoding)
convert_numbers.py:16:8: W0702: No exception type(s) specified (bare-except)
convert_numbers.py:23:0: C0116: Missing function or method docstring (missing-function-docstring)
convert_numbers.py:23:24: W0621: Redefining name 'num' from outer scope (line 53) (redefined-outer-name)
convert_numbers.py:24:2: W0621: Redefining name 'binario' from outer scope (line 54) (redefined-outer-name)
convert_numbers.py:30:0: C0116: Missing function or method docstring (missing-function-docstring)
convert_numbers.py:30:28: W0621: Redefining name 'num' from outer scope (line 53) (redefined-outer-name)
convert_numbers.py:32:2: W0621: Redefining name 'hexadecimal' from outer scope (line 55) (redefined-outer-name)
convert_numbers.py:54:4: C0103: Constant name "binario" doesn't conform to UPPER_CASE naming style (invalid-name)
convert_numbers.py:55:4: C0103: Constant name "hexadecimal" doesn't conform to UPPER_CASE naming style (invalid-name)
convert_numbers.py:63:7: W1514: Using open without explicitly specifying an encoding (unspecified-encoding)

-----------------------------------
Your code has been rated at 0.00/10

